packages/db-migrations/databases/fxa/patches/patch-136-137.sql

```diff
@@ -0,0 +1,7 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('136');
+
+INSERT INTO securityEventNames(name) VALUES ('account.login.failure');
```

Comment:
Should there be a separate PR with all the new event names? Otherwise we might end up with lots of patches.

Reply:
For sure, this was just a first cut to see if I could get it working. I'll add them in.